act-rules-format.bs

```diff
@@ -15,86 +15,112 @@ Markup Shorthands: markdown yes
 Introduction {#intro}
 =====================
 
-There are currently many tools available which aid their users in testing web content for conformance to accessibility standards such as the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [[WCAG]]. As the web develops in both size and complexity, these tools are essential for managing the resources available on the web.
+There are currently many test procedures and tools available which aid their users in testing web content for conformance to accessibility standards such as the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [[WCAG]]. As the web develops in both size and complexity, these procedures and tools are essential for managing the accessibility of resources available on the web.
 
 This format is intended to enable a consistent interpretation of how to test for [accessibility requirements](#structure-accessibility-requirements) to avoid conflicting results of accessibility tests. It is intended to be used to describe both manual accessibility tests as well as automated tests performed by accessibility testing tools (ATTs).
```

Comment:
`to avoid conflicting results of accessibility tests` makes it sound negative straight up, that there is a chance of negative interpretation.

Rather leave it with a positive note?

`This format is intended to enable a consistent interpretation of how to test for accessibility requirements.`